### PR TITLE
16 entropy calculators

### PR DESCRIPTION
**Pull request by Coop8**
Entropy time!

### CODE REVIEW

File: include/Utility.hpp [hunk 1]
```diff
@@ -39,20 +39,18 @@ template <typename T> std::pair<T, size_t> minimum(const std::vector<T> &vec)
 
 double convert(const std::string &str);
 
-template <typename T> double ClosestSamplingPoint(const std::vector<T> &vec, T const &val)
+inline double ClosestSamplingPoint(const std::vector<double> &vec, double const &val)
```
Comment (galaxybomb23) [inline include/Utility.hpp:42]: what does inline do?

File: test/EntropyCalculatorsTest.cpp
Comment (galaxybomb23) [inline test/EntropyCalculatorsTest.cpp:1]: what is the point of this file?

Comment (Coop8) [reply test/EntropyCalculatorsTest.cpp:1]: not sure, removed

File: include/Utility.hpp (hunk 1 above)
Comment (Coop8) [reply include/Utility.hpp:42]: it replaces the function call with the code itself, needed to either make it inline due to compiler errors of multiple definitions across files where that wasn't true.

**Review by iBoot32 — APPROVED** [review]
lgtm

**Pull request merged**